saleor/account/models.py

```diff
@@ -189,6 +191,48 @@ def get_ajax_label(self):
         return self.email
 
 
+class Bot(models.Model):
```

Comment:
We might want to allow metadata for bots? That would be useful to add some data such as urls, descriptions, etc.